examples/pipelines/simple_tutorial/TUTORIAL.md

```diff
@@ -0,0 +1,48 @@
+# bionode-watermill for dummies!
+
+## Why?
+
+This tutorial is intended for those that attempt to assemble a bioinformatics 
+pipeline using bionode-watermill for the first time. 
+
+## Defining a task
+
+Watermill is a tool that lets you orchestrate tasks. So, lets first 
+understand how to define a task. 
+ 
+To define a task we first need to require bionode-watermill:
+
+```javascript
+const watermill = require('../../..') /* don't forget to edit this require if
+ you run it outside bionode-watermill examples/pipelines/simple_tutorial 
+ folder */ 
+const task = watermill.task  /* have to specify task since bionode-watermill 
+has more operators*/
+```
+
+After, we can use task variable to define a task:
+
+```javascript
+// this is a kiss example of how tasks work with shell
+const simpleTask = task({
+  params: 'Test_file.txt',  /*defines parameters to be passed to the
+     task function*/
+  name: 'This is the task name' //defines the name of the task
+}, ({ params }) => `touch ${params}`
```

Comment:
I think you have a point. ES6 is not standard js, so we should exemplify it first with standard js.